tests/e2e/e2e_test.go

```diff
@@ -73,6 +73,29 @@ func pingSvc(url string) {
 	}
 }
 
+func pingCmd(cmd string) bool {
+	pingTimeout := time.After(5 * time.Minute)
+	tick := time.Tick(time.Second)
+
+	for {
+		select {
+		case <-pingTimeout:
+			Fail("timeout out after 5 minutes")
+
+		case <-tick:
+			out, err := exec.Command("/bin/sh", "-c", cmd).Output()
+			if err != nil {
+				Fail(err.Error())
+			}
+
+			outInt, _ := strconv.Atoi(string(out))
```

Comment:
why are we converting the output to int?

Reply:
@kadel I need the output in int since I am doing `wc -l` comparison

Reply:
then we need a different name for this function. It is really specific one command, and it won't work for anything that is not ending with `wc`

Reply:
:+1: that looks better. 
It might be good to change function name `pingCmd` doesn't really indicate what this function does. 
Also, don't forget to add a comment with a description what this function is for.